lib/src/revset.rs

```diff
@@ -755,6 +775,22 @@ pub fn expect_string_pattern(node: &ExpressionNode) -> Result<StringPattern, Rev
     revset_parser::expect_pattern_with("string pattern", node, parse_pattern)
 }
 
+fn expect_timeexpression<Tz: TimeZone>(
```

Comment:
nit: `expect_time_expression` (missing underscore)?

Reply:
Underscore added to all the function names I added.